cmd/filterGenotypes/filterGenotypes.go

```diff
@@ -1,3 +1,5 @@
+//filter genotyped VCFs containing at least 3 samples with SNP regions where parental genomes are homozygous (and different) and have a clear heterozygous F1 from the parents
```

Comment:
Maybe there should be a space before filter and it should begin with a capital F